src/components/RatingChart/RatingChart.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Bar } from 'react-chartjs-2';
+import Loader from '../Loader/Loader';
+import styles from './RatingHistory.module.css';
```

Comment:
This file doesn't even exist. Did u forget to commit?.How did you define your className with `styles` when it did not even exist? :disappointed: